src/controllers/autoCompletion.controller.js

```diff
@@ -73,3 +90,23 @@ exports.getAutoCompletions = async function (req, res, next) {
     });
   }
 };
+
+exports.deleteAutoCompletions = async function (req, res, next) {
+  const { history } = req.query;
+  const { accessToken } = req.cookies;
+  const decodedToken = jwt.decode(accessToken);
+  const { userId } = decodedToken;
+
+  const user = await User.findOne({
+    _id: userId,
+  });
```

Comment:
```suggestion
  const user = await User.findById(userId);
```
로 수정해도 좋을 것 같습니다!

조사해보니 두 함수 모두 findOne()을 실행하는데, 
findOne(undefined)은 findOne({})과 똑같고 임의의 도큐먼트를 반환하고
반면에 findById(undefined)는 findById({ _id: null })과 같이 동작한다고 합니다.
[findById vs findOne](https://stackoverflow.com/questions/14255603/what-is-the-difference-between-model-findone-model-findbyid-in-mongoose)
[findOne](https://mongoosejs.com/docs/api/model.html#Model.findOne())

Reply:
그렇군요! 수정하겠습니다!